### ISSUE TITLE
Unable to login to scrapbox with sbe on macOS (security issue)

### ISSUE THREAD

**Issue by vitalibondar** [kondoumh/sbe#8]
Google blocks all login attempts using sbe on macOS Catalina
![CleanShot 2020-05-09 at 01 38 42@2x](https://user-images.githubusercontent.com/10779590/81455381-be7a5980-9197-11ea-81c3-be8eb76a855c.jpeg)

**Comment by kondoumh** [comment]
Thanks for the report, I'm guessing it's an issue with your Google account security settings. I don't know how to set it up, unfortunately.

https://support.google.com/accounts/thread/22873505?hl=en

**Comment by vitalibondar** [comment]
> I'm guessing it's an issue with your Google account security settings. I don't know how to set it up, unfortunately.
> https://support.google.com/accounts/thread/22873505?hl=en
Yes, you're right. Two-factor authentication is enabled for my account. In such cases, you can log in to your account with the application password. I generated this password, but I can't log in with it, because after entering my email, Google immediately redirects me to the page shown in the screenshot above.

**Comment by vitalibondar** [comment]
I disabled two-factor authentication, enabled the Less Secure Apps account login feature. But I still can't log in to the Scrapbox from sbe.
This link https://support.google.com/accounts/answer/7675428?hl=en lists the reasons why authentication is not successful:

> To help protect your account, Google doesn't let you sign in from some browsers. Google may stop sign-ins from browsers that:
> 
> - Don't support JavaScript or have Javascript turned off.
> - Have unsecure or unsupported extensions added.
> - Use automation testing frameworks.
> - Are embedded in a different application.

**Comment by ciceronianus** [comment]
Hi, I have the very same problem :(. Hopefully, somebody will be able to find workaround.
